ida-customizations/ida-course/0-blockchain-basics/3-managed.md

```diff
@@ -69,29 +69,9 @@ The protocol defines the **Gas cost** for each operation in the EVM. The halting
 
 In Ethereum, **smart contracts** are stored on every single node - they are public. In practice, a smart contract in the EVM is an autonomous agent with an internal account. The most popular language for writing such a contract is **Solidity**.
 
-<ExpansionPanel title="Faster block time">
+Ethereum recently transitioned from Proof-of-Work to Proof-of-Stake consensus. Block times were unaffected by this change, by design, and are much faster - around 14 seconds at the time of writing.
 
-Ethereum has a much faster block time than Bitcoin - currently it is around 15 seconds.
-
-Ethereum implements a variation of Bitcoin's PoW consensus algorithm called Ethash, which is intended to be [ASIC-resistant and GPU friendly](https://github.com/ethereum/wiki/wiki/Ethash-Design-Rationale). ASIC-resistance is meant to discourage unwanted concentrations of mining power; in other words, to encourage decentralization. Core developers of the Ethereum project plan to switch to the **proof-of-stake (PoS)** consensus algorithm in the future.
```

Comment:
Should we put this in an expandable view for the curious only?

Reply:
I thought about it because GHOST is a clever innovation and a better PoW. I decided to remove it because PoW isn't used by many (any?) new chains ,incl. Cosmos. I thought the relevance is questionable, so inclined to keeping the focus on things one needs to know to understand Cosmos. "Eth has faster blocks" is still true and still relevant as well as the notion of general-purpose usage. 

This is a easy to put back if there's a strong argument for unfolding techniques Eth used to improve the consensus they don't use anymore.